src/button/button-contributions.ts

```diff
@@ -249,23 +249,10 @@ export const buttonContributions: ButtonContributionParams[] = [
         id: "gl-issue",
         exampleUrls: ["https://gitlab.com/svenefftinge/browser-extension-test/-/issues/1"],
         match: /\/issues\//,
-        selector:
-            "#content-body > div.issue-details.issuable-details.js-issue-details > div.detail-page-description.content-block.js-detail-page-description.gl-pt-3.gl-pb-0.gl-border-none > div:nth-child(1) > div > div.gl-flex.gl-items-start.gl-flex-col.md\\:gl-flex-row.gl-gap-3.gl-pt-3 > div",
+        selector: "xpath://*[@id='content-body']/div[2]/div/section/section/div[2]/div[1]/div[2]",
```

Comment:
L252 applies to existing issues, not newly created issues.

Perhaps we can make a duplicate entry, but with a slightly different selector?

Ref: https://linear.app/gitpod/issue/NEXT-4008/browser-extension-open-button-missing-on-pull-request-page-for#comment-05536ac1